core/src/Microsoft.Identity.Core.csproj

```diff
@@ -19,17 +11,9 @@
     <CodeAnalysisRuleSet>../ConfigureAwait.ruleset</CodeAnalysisRuleSet>
 </PropertyGroup>
 
-  <PropertyGroup Label="GenerateAssemblyInfo inputs">
-    <!-- This should be passed from the VSTS build -->
-    <CoreSemVer Condition="'$(CoreSemVer)' == '1.0.0-privatebuild'"></CoreSemVer>
-
-    <!-- This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion -->
-    <Version>$(CoreSemVer)</Version>
-
-    <Copyright>Copyright (c) Microsoft Corporation. All rights reserved.</Copyright>
-    <Company>Microsoft Corporation</Company>
-  </PropertyGroup>
-
+<!-- Important note: this assembly should not be shipped to customers; it acts only as a placeholder for files that are included
+in ADAL and MSAL and to ease unit testing. No projects should take a dependency on Core.dll either -->
+  
```

Comment:
Is there a way to ensure that no project take a dependency on core.dll?

Reply:
Layer2 analyzer? We could move all core unit tests to either ADAL or MSAL (or both) and make core.dll either a shared project or a .net standard 4.7 project, on which ADAL and MSAL cannot take a reference.